cardinal/option.go

```diff
@@ -84,18 +83,12 @@ func WithStoreManager(s gamestate.Manager) WorldOption {
 	}
 }
 
-func WithEventHub(eventHub events.EventHub) WorldOption {
+func WithEventHub(eventHub *events.EventHub) WorldOption {
```

Comment:
since we no longer have multiple event hubs, does it need to be an optional still?

Reply:
good point, i'll nuke this